Drafted a post for EGD2025 (#3389)
galaxyproject/galaxy-hub: content/news/2025-10-15-egd2025/index.md

Slides & photos
  - Photo gallery: https://photos.app.goo.gl/haiWYYBryV4pKvxe9
  - Slides: 
  | Slide Title | Presenter(s) | Link |
| --- | --- | --- |
| Welcome, Logistics & General Galaxy and ELIXIR Update | Hans-Rudolf Hotz | https://f1000research.com/slides/14-1066 |
| Special Interest Groups and Working Groups: the Galaxy Governance Model | Marisa Loach | https://f1000research.com/slides/14-1067 |
| The Microbiology Galaxy Lab and the effort from the microGalaxy to build the first community-driven gateway for reproducible and FAIR analysis of microbial data | Bérénice Batut | https://f1000research.com/slides/14-1021 |

> | Building More Usable Galaxy Workflows with Improved Dataset Collections | John Chilton, Marius van den Beek, Ahmed Awan | https://f1000research.com/slides/14-1069 |

Author: ```suggestion
| Building More Usable Galaxy Workflows with Improved Dataset Collections | John Chilton | https://f1000research.com/slides/14-1069 |
```

| Building More Usable Galaxy Workflows with Improved Dataset Collections | Mina Hojat Ansari, Paul Zierep, Judith Korb | https://f1000research.com/slides/14-1070 |

> | Managing materials science workflows with Galaxy at STFC | Leandro Liborio, Patrick Austin, Subindev Devadasanet al | https://f1000research.com/slides/14-1071 |

Author: ```suggestion
| Managing materials science workflows with Galaxy at STFC | Leandro Liborio | https://f1000research.com/slides/14-1071 |
```

| The ISCC Discovery Protocol: Decentralized signing, timestamping, and discovery for the International Standard Content Code (ISCC) | Titusz Pan | https://f1000research.com/slides/14-1072 |

> | The Galaxy Community in Switzerland | Hans-Rudolf Hotz, Lucille Delisle, Vipin T. Sreedharan | https://f1000research.com/slides/14-1073 |

Author: ```suggestion
| The Galaxy Community in Switzerland | Hans-Rudolf Hotz | https://f1000research.com/slides/14-1073 |
```

| Evolution of Galaxy at Bologna Sant’Orsola Hospital | Giuseppe Profiti | https://f1000research.com/slides/14-1085 |
| Precision Diagnosis and Risk Stratification of Helicobacter pylori-Induced Gastritis Using Multi-Omics Machine Learning Signatures Or The Synergy Between Complex Research Projects and Galaxy | Paul Zierep, Georg Häcker, Rolf Backofen | https://f1000research.com/slides/14-1086 |
| Broadening the Galaxy: Introducing Digital Humanities in Galaxy | Daniela Schneider | https://f1000research.com/slides/14-1087 |
| Serving Locally Hosted LLMs with Galaxy | Arash Kadkhodaei | https://f1000research.com/slides/14-1088 |
| pyGenomeTracks – make a publishable figure out of your NGS data | Lucille Lopez-Delisle | https://f1000research.com/slides/14-1089 |
| Galaxy Ecology: A new "2 step approach" to contribute source code for Galaxy (re)users(From raw data to operational indicators through essential variables) | Yvan Le Bras, Arhur Barreau, Marie Josséet al | https://f1000research.com/slides/14-1090 |
| Galaxy Tools 2.0 — The next step in accessibility? | Marius van den Beek | https://f1000research.com/slides/14-1091 |

> | Simple and Secure Credential Handling for Tools in Galaxy | Alireza Heidari, Arash Kadkhodaei, David Lopezet al | https://f1000research.com/slides/14-1092 |

Author: ```suggestion
| Simple and Secure Credential Handling for Tools in Galaxy | Alireza Heidari | https://f1000research.com/slides/14-1092 |
```

| New integrations: ELNs (eLabFTW, RSpace) and ARC job runner | José Manuel Domínguez | https://f1000research.com/slides/14-1093 |
| Making Galaxy workflows sustainable and FAIR | Nicola Soranzo | https://f1000research.com/slides/14-1094 |
| Galaxy Ecology 2025 update | Yvan Le Bras | https://f1000research.com/slides/14-1095 |

> | Galaxy facing EARTH SYSTEM challenges | Jérôme Detoc, Marie Jossé, Pauline Seguineauet al | https://f1000research.com/slides/14-1096 |

Author: ```suggestion
| Galaxy facing EARTH SYSTEM challenges | Jérôme Detoc | https://f1000research.com/slides/14-1096 |
```

---
title: "European Galaxy Days 2025: Many communities, one buzzing week in Freiburg"
date: "2025-10-15"
authors: "Armin Dadras, Björn Grüning, and Hans-Rudolf Hotz"
tease: "1–3 Oct 2025, Freiburg: European Galaxy Days, ELIXIR Galaxy, and EuroScienceGateway members convened to align roadmaps and turn plans into solutions, tools, and workflows through talks, demos, and a CoFest."
tags: [eosc, meeting, esg, esg-wp1, esg-wp2, esg-wp3, esg-wp4, esg-wp5, conference, egd]
supporters: [eosc, unifreiburg, esg, elixir, eu]
subsites: [all, esg]
main_subsite: eu
---

> From **1–3 October 2025**, the [University of Freiburg](https://uni-freiburg.de/en/) hosted a packed, three-in-one meeting: **European Galaxy Days (EGD)**, the **[ELIXIR Galaxy Community](https://elixir-europe.org/communities/galaxy) face-to-face meeting**, and the **[EuroScienceGateway (ESG)](https://galaxyproject.org/projects/esg/) Final Meeting** with ~100 participants (87 in person + 12 online). Running side-by-side, these events created a special week where platform maintainers, tool and workflow developers, and domain scientists aligned roadmaps and moved from slides to practical solutions and future collaborations.

Author: ```suggestion
From **1–3 October 2025**, the [University of Freiburg](https://uni-freiburg.de/en/) hosted a packed, three-in-one meeting: [**European Galaxy Days (EGD)**,](https://galaxyproject.org/events/2025-10-01-egd2025/) the **[ELIXIR Galaxy Community](https://elixir-europe.org/communities/galaxy) face-to-face meeting**, and the **[EuroScienceGateway (ESG)](https://galaxyproject.org/projects/esg/) Final Meeting** with ~100 participants (87 in person + 12 online). Running side-by-side, these events created a special week where platform maintainers, tool and workflow developers, and domain scientists aligned roadmaps and moved from slides to practical solutions and future collaborations.
```

> Across talks, demos, Birds-of-a-Feathers (BoFs), and a concluding hackathon, the week showed how the Galaxy ecosystem is advancing research across domains from [Microbiology](https://microbiology.usegalaxy.org/), [Materials Science](https://materials.usegalaxy.eu/), [Ecology](https://ecology.usegalaxy.eu/), [Climate Science](https://climate.usegalaxy.eu/), [Imaging](https://imaging.usegalaxy.eu/), and [Digital Humanities](https://galaxyproject.org/community/sig/digital-humanities/). Freiburg’s campus hosted hybrid sessions and BoFs across multiple rooms, with Friday dedicated to a focused Co-Fest & Hackathon where groups squashed bugs, improved tools, tuned workflows, and formed plans for next collaborations. Coverage rolled out on [Mastodon](https://mastodon.social/tags/EGD2025), [Bluesky](https://bsky.app/hashtag/EGD2025?author=galaxyproject.bsky.social), and [LinkedIn](https://www.linkedin.com/search/results/all/?keywords=%23egd2025&origin=HASH_TAG_FROM_FEED&sid=kCg) (47 posts with #EGD2025), while Matrix channels ([usegalaxy-eu Lobby](https://matrix.to/#/#usegalaxy-eu_Lobby:gitter.im) and [use-galaxy announce](https://matrix.to/#/!ArjKhGljVCmzqxhauY:matrix.org?via=matrix.org&via=matrix.uni-freiburg.de&via=t2bot.io)) supported live coordination and announcements.

Author: ```suggestion
Across talks, demos, Birds-of-a-Feathers (BoFs), and a concluding hackathon, the week showed how the Galaxy ecosystem is advancing research across domains from [Microbiology](https://microbiology.usegalaxy.org/), [Materials Science](https://materials.usegalaxy.eu/), [Ecology](https://ecology.usegalaxy.eu/), [Climate Science](https://climate.usegalaxy.eu/), [Imaging](https://imaging.usegalaxy.eu/), and [Digital Humanities](https://galaxyproject.org/community/sig/digital-humanities/). Freiburg’s campus hosted hybrid sessions and BoFs across multiple rooms, with Friday dedicated to a focused Co-Fest & Hackathon where groups squashed bugs, improved tools, tuned workflows, and formed plans for next collaborations. If you want to learn more about the event, please have a look at our social media coverage on [Mastodon](https://mastodon.social/tags/EGD2025), [Bluesky](https://bsky.app/hashtag/EGD2025?author=galaxyproject.bsky.social), and [LinkedIn](https://www.linkedin.com/search/results/all/?keywords=%23egd2025&origin=HASH_TAG_FROM_FEED&sid=kCg) (47 posts with #EGD2025). Also, Matrix channels ([usegalaxy-eu Lobby](https://matrix.to/#/#usegalaxy-eu_Lobby:gitter.im) and [use-galaxy announce](https://matrix.to/#/!ArjKhGljVCmzqxhauY:matrix.org?via=matrix.org&via=matrix.uni-freiburg.de&via=t2bot.io)) supported live coordination and announcements and both are still available if you are interested to join.
```

Slides & photos

> <!--- To do by Armin until next week --->

Author: ```suggestion
```

  - Slides: 
  | Slide Title | Presenter(s) | Link |
| --- | --- | --- |

> | Welcome, Logistics & General Galaxy and ELIXIR Update | Hans-Rudolf Hotz | https://f1000research.com/slides/14-1066 |

Author: Done.
